src/screens/Main.tsx

```diff
@@ -35,8 +36,23 @@ interface Props {
 }
 
 export function Main({ className }: Props) {
+  const { extensionExists, accountExists } = useKeyringContext();
   const { sourceChainDetails, targetChainDetails } = useSourceTarget();
 
+  const proj = [];
```

Comment:
i would extract this logic to a separate component so is easier as well to apply designs

Reply:
I will not do that at this point - Already talked with @goldsteinsveta and agreed to prepare the ground until Figma is ready - and then we will start putting in the correct position.